<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gaalop Compiler Driver Configuration Tool</string>
  </property>
  <widget class="QPushButton" name="selectMapleBinaryDir">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveSettings">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>521</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectCppCompileCommand">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectCudaCompileCommand">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>70</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Maple Binary Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>C++ Command</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>CUDA Command</string>
   </property>
  </widget>
  <widget class="QComboBox" name="mapleBinaryDir">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cppCompileCommand">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="cudaCompileCommand">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Java Compile Command</string>
   </property>
  </widget>
  <widget class="QComboBox" name="javaCompileCommand">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>351</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="selectJavaCompileCommand">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>100</y>
     <width>21</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
